pimd/pim6_cmd.c

```diff
@@ -1569,10 +1576,250 @@ DEFPY (show_ipv6_pim_nexthop_lookup,
 	return CMD_SUCCESS;
 }
 
+DEFUN (debug_pimv6,
+       debug_pimv6_cmd,
+       "debug pimv6",
+       DEBUG_STR
+       DEBUG_PIMV6_STR)
+{
+	return pim_debug_pim_cmd();
+}
+
+DEFUN (no_debug_pimv6,
+       no_debug_pimv6_cmd,
```

Comment:
You could shorten the code by combining the positive and negative debug commands into `[no] debug pimv6 …`. `DEFPY` will automatically create a `no` argument for `[no]` in the beginning. (No need for `[no$no]`.)

Reply:
Fixed